CodePush.js

```diff
@@ -1,15 +1,13 @@
 'use strict';
 
-var { Alert } = require("./AlertAdapter");
-var NativeCodePush = require("react-native").NativeModules.CodePush;
-var PackageMixins = require("./package-mixins")(NativeCodePush);
-var requestFetchAdapter = require("./request-fetch-adapter.js");
-var Sdk = require("code-push/script/acquisition-sdk").AcquisitionManager;
-var semver = require("semver");
+import { Alert } from "./AlertAdapter";
+const NativeCodePush = require("react-native").NativeModules.CodePush;
+const PackageMixins = require("./package-mixins")(NativeCodePush);
+import requestFetchAdapter from "./request-fetch-adapter.js";
+import { AcquisitionManager as Sdk } from "code-push/script/acquisition-sdk";
```

Comment:
Should we move this to the top of the import "block"? I'd love to organize these import statements a little bit. I'd also be in favor of seperate to the imports from the const/requires.